package.json

```diff
@@ -50,7 +50,8 @@
 		"katex": "^0.16.9",
 		"marked": "^9.1.0",
 		"svelte-french-toast": "^1.2.0",
+		"sveltekit-i18n": "^2.4.2",
```

Comment:
Thanks for the PR! `sveltekit-i18n` seems like it's no longer maintained, is there a way to use framework agnostic `i18n` library?

Reply:
`iI18next` seems like a good suggestion.

We can explore `i18next` further or research other framework agnostic solutions, @tjbck  Do you have any specific solution in mind?

Reply:
`i18next` looks good to me as well!

Reply:
I will investigate `i18next` and attempt to create an initial configuration for it.

Reply:
@tjbck Basic i18next setup is done, I hope adding `i18next-browser-languagedetector` and `i18next-resources-to-backend` is ok. Please let me know if these additions are not acceptable or if there's a preferred alternative approach that should be taken instead. 
Next step can be scanning the whole project for string literals to be added to json locale files.

Reply:
@asedmammad Thanks for all the effort! They both seem promising but are there any alternatives that I should be aware of?

Reply:
@tjbck My pleasure!
Manually detecting the active language using localStorage, query param, and `window.navigator.language` is an alternative approach.
Other i18next language detector plugins can be found here: https://www.i18next.com/overview/plugins-and-utils#language-detector

Having translations in project json files seems a reasonably good option, though adds a few KBs to the build. If the bundle size is/becomes a concern, we could split the chunks to avoid loading all translation files at first load.  I will compare the bundle size against the latest `dev` commit and update here.
Another option is to load translations over http from the project's backend. 
i18next provides some translation backend providers here: 
https://www.i18next.com/overview/plugins-and-utils#backends